src/vs/workbench/workbench.common.main.ts

```diff
@@ -395,4 +394,8 @@ import 'vs/workbench/contrib/accountEntitlements/browser/accountsEntitlements.co
 
 // Synchronized Scrolling
 import 'vs/workbench/contrib/scrollLocking/browser/scrollLocking.contribution';
+
+// Issues
+import 'vs/workbench/contrib/issue/browser/issueTroubleshoot';
```

Comment:
I think we can actually remove this line by having it in `common/issue.contribution.ts` which seems to be imported in both web and desktop.

Reply:
@justschen you still need to add it to `common/issue.contribution.ts` !

Reply:
@bpasero looks like that import isn't allowed in `workbench/contrib/issue/browser`

Reply:
@justschen ah yes, in that case I suggest to add it in both `issue.contributions`